sdk/python/kfp/dsl/types.py

```diff
@@ -17,187 +17,215 @@
+        return False
+  return True
+
 
 def _instance_to_dict(instance):
-	'''_instance_to_dict serializes the type instance into a python dictionary
+  '''_instance_to_dict serializes the type instance into a python dictionary
 	Args:
 		instance(BaseType): An instance that describes a type
 
 	Return:
 		dict
 	'''
-	return {type(instance).__name__: instance.__dict__}
+  return {type(instance).__name__: instance.__dict__}
+
 
 def _check_dict_types(checked_type, expected_type):
-	'''_check_dict_types checks the type consistency.
-	Args:
-  	checked_type (dict): A dict that describes a type from the upstream component output
-  	expected_type (dict): A dict that describes a type from the downstream component input
-	'''
-	if not checked_type or not expected_type:
-		# If the type is empty, it matches any types
-		return True
-	checked_type_name,_ = list(checked_type.items())[0]
-	expected_type_name,_ = list(expected_type.items())[0]
-	if checked_type_name == '' or expected_type_name == '':
-		# If the type name is empty, it matches any types
-		return True
-	if checked_type_name != expected_type_name:
-		print('type name ' + str(checked_type_name) + ' is different from expected: ' + str(expected_type_name))
-		return False
-	type_name = checked_type_name
-	for type_property in checked_type[type_name]:
-		if type_property not in expected_type[type_name]:
-			print(type_name + ' has a property ' + str(type_property) + ' that the latter does not.')
-			return False
-		if checked_type[type_name][type_property] != expected_type[type_name][type_property]:
-			print(type_name + ' has a property ' + str(type_property) + ' with value: ' +
-						str(checked_type[type_name][type_property]) + ' and ' +
-						str(expected_type[type_name][type_property]))
-			return False
-	return True
+  """_check_dict_types checks the type consistency.
+
+  Args:
+  checked_type (dict): A dict that describes a type from the upstream component output
+  expected_type (dict): A dict that describes a type from the downstream component input
+
+  """
+  if not checked_type or not expected_type:
+    # If the type is empty, it matches any types
+    return True
+  checked_type_name, _ = list(checked_type.items())[0]
+  expected_type_name, _ = list(expected_type.items())[0]
+  if checked_type_name == '' or expected_type_name == '':
+    # If the type name is empty, it matches any types
+    return True
+  if checked_type_name != expected_type_name:
+    print(
+        'type name ' + str(checked_type_name) +
+        ' is different from expected: ' + str(expected_type_name)
+    )
+    return False
+  type_name = checked_type_name
+  for type_property in checked_type[type_name]:
+    if type_property not in expected_type[type_name]:
+      print(
+          type_name + ' has a property ' + str(type_property) +
+          ' that the latter does not.'
+      )
+      return False
+    if checked_type[type_name][type_property] != expected_type[type_name
```

Comment:
I think that this line looks a little strange.

Reply:
Agree. Auto-format is not perfect.